DXMainClient/DXGUI/Multiplayer/CnCNet/LoadOrSaveGameOptionPresetWindow.cs

```diff
@@ -132,9 +132,10 @@ public LoadOrSaveGameOptionPresetWindow(WindowManager windowManager) : base(wind
 
         public override void Initialize()
         {
+            Name = "LoadOrSaveGameOptionPresetWindow";
```

Comment:
```suggestion
            Name = nameof(LoadOrSaveGameOptionPresetWindow);
```

Reply:
> ```suggestion
> 
>             Name = nameof(LoadOrSaveGameOptionPresetWindow);
> 
> ```

https://github.com/CnCNet/xna-cncnet-client/pull/721#issuecomment-2910571989